common/fdlimit/doc.go

```diff
@@ -14,5 +14,10 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package fdlimit handles file descriptors based on the operating system.
+/*
+Package fdlimit handles file descriptors based on the operating system
+
+In Klaytn, `leveldb` database in the `database` package is using this package to get current allowance of file descriptors for this process and to raise it as it requires.
```

Comment:
what about this?
In Klaytn, `leveldb` database in the `database` package uses this package ~~
or
In Klaytn, Package fdlimit is used in `database` package to get current allowance ~~

Reply:
@laggu Actually it was to specify that `leveldb` is using it. There can be other databases but others are not using the package.